Source/WebCore/html/shadow/DateTimeFieldElements.cpp

```diff
@@ -49,6 +51,8 @@ Ref<DateTimeDayFieldElement> DateTimeDayFieldElement::create(Document& document,
     static MainThreadNeverDestroyed<const AtomString> dayPseudoId("-webkit-datetime-edit-day-field"_s);
     ScriptDisallowedScope::EventAllowedScope eventAllowedScope { element };
     element->initialize(dayPseudoId);
+    element->setAttributeWithoutSynchronization(HTMLNames::roleAttr, AtomString { "spinbutton"_s });
```

Comment:
Can we get the "spinbutton" string using `accessibilityRoleToString` from `AccessibilityObjectInterface.h`?